fastlane/Fastfile

```diff
@@ -130,7 +130,8 @@ platform :ios do
   desc 'Run the unit tests'
   lane :test do
     run_tests(
-      scheme: SCHEME
+      scheme: SCHEME,
+      xcargs: "-skipPackagePluginValidation"
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/2daecc5ee5662b7c036d698dfd74684833a0d748/fastlane/Fastfile#L134"></span>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>